t/lib/OpenQA/Test/FullstackUtils.pm

```diff
@@ -123,7 +123,7 @@ sub wait_for_developer_console_like ($driver, $message_regex, $diag_info) {
     my $previous_log = '';
     # poll less frequently when waiting for paused (might take a minute for the first test module to pass)
     my $check_interval = $diag_info eq 'paused' ? 5 : 1;
-    my $timeout = OpenQA::Test::TimeLimit::scale_timeout(ONE_MINUTE * 5) * $check_interval;
+    my $timeout = OpenQA::Test::TimeLimit::scale_timeout($timeout_s);
```

Comment:
I guess based on our collaborative poking we assume that 2 minutes is enough for any successful run.

Reply:
yes, should be